package.json

```diff
@@ -20,7 +21,8 @@
   },
   "homepage": "https://github.com/IgnacioBarocchi/HumbleScraper#readme",
   "dependencies": {
-    "node-fetch": "^2.6.0"
+    "node-fetch": "^2.6.0",
+    "yargs": "^15.3.1"
```

Comment:
get rid of `yargs`, remove it from here and run
```
npm -rf node_modules
npm i
```
to re-generate package-lock.json